### ISSUE TITLE
Validation error when including rfc2161 datetime values in included json examples

### ISSUE THREAD

**Issue by howzat** [mulesoft/api-workbench#329]
Reading through the docs, I think this should work. Given the following two files (test.raml and test.json) the workbench reports "Valid rfc2616 formatted string is expected".  Thanks!

test.raml
```
#%RAML 1.0 Library
usage: |
  Exposition
types:

  Test:
    type: object
    properties:
      t:
        type: datetime
        format: rfc2616
     example: !include test.json
```
test.json
```
{
  "t": "Sat, 8 Oct 2016 20:04:37 GMT" // this is a valid rfc2616 datetime
}
```
[Archive.zip](https://github.com/mulesoft/api-workbench/files/518398/Archive.zip)

**Comment by howzat** [comment]
Apologies, I'm not sure this is an issue, but it's certainly worth recording for posterity. The issue appears to be whether the datetime uses 1 or 2 digits for the day for example:

"Sat, **8** Oct 2016 20:04:37 GMT"
"Sat, **08** Oct 2016 20:04:37 GMT"

I generated my datetime using the Java [RFC_1123_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#RFC_1123_DATE_TIME) which I'd assumed would be compatible with rfc2616. [RFC1123](https://www.ietf.org/rfc/rfc1123.txt) updates RFC2616 to allow 1 **or** 2 digits. 

In short, datetimes must be rf2616 compliant, rfc1123 are not supported. This might be obvious in retrospect :(, but I spent some time working it out, I hope this is useful for someone else.
